themes/salix/layouts/partials/header.html

```diff
@@ -3,6 +3,9 @@
 </a>
 <header>
 	<h1>{{ replace (.Site.Title | plainify) " " "<br/>" | safeHTML}}.</h1>
+	<div>
+		<img src="./img/willowtexture.png" alt="close up of willow bark"/>
```

Comment:
I don't know if we need an alt text here; let's just hide this from screen readers. It's a background, not important content.

Reply:
I've added `aria-hidden="true"`. I'm not sure if that means that the alt text is now completely redundant or there might be some way for people to access it still. I've taken the cautionary approach and left it in.